crds/prometheus-operator/Readme.md

```diff
@@ -0,0 +1,20 @@
+# How to use vcluster-generic-crd-sync-plugin to sync Prometheus Operator resources
+**Note**: Currently this plugin is still underdevelopment and is provided without any commercial support. You are encouraged to report problems via Issues in this repo. For the help with troubleshooting you can post to the "vcluster" channel in our Slack - https://slack.loft.sh/  
+**Note**: The configuration provided here is an example. Please review and update it to match your environment and use case.
+
+
+Prerequisite for installation are the Prometheus CRDs installed in the host cluster where vcluster will be installed.
+
+
+To install this plugin you need to pass the ./plugin.yaml file as additional source of Helm values during the installation. This is described in more detail in vcluster docs - https://www.vcluster.com/docs/plugins/overview#loading-and-installing-plugins-to-vcluster  
+For testing purposes you can refer to the plugin.yaml for the Prometheus Operator resources like so: `https://raw.githubusercontent.com/loft-sh/vcluster-generic-crd-sync-plugin/main/crds/prometheus-operator/plugin.yaml`.  
+Example installation command:
+```
+vcluster create -n vcluster vcluster -f https://raw.githubusercontent.com/loft-sh/vcluster-generic-crd-sync-plugin/main/crds/prometheus-operator/plugin.yaml
```

Comment:
Use this URL until the PR is merged:
https://prometheus-operator.dev/docs/operator/api/#monitoring.coreos.com/v1.ServiceMonitor